app/views/base/layout.scala

```diff
@@ -416,7 +416,10 @@ object layout:
           ),
           ctx.blind option h2("Navigation"),
           !ctx.isAppealUser option frag(
-            topnav(),
+             ctx.me map { me =>
+             topnav(me)
+            },
```

Comment:
this removes the entire top nav menu for users that are not logged in. Also preventing them from logging in.

Reply:
Got it!